packages/jest/src/index.js

```diff
@@ -42,6 +43,10 @@ function normalizeJestOptions(opts, neutrino, usingBabel) {
     ...(opts.moduleFileExtensions || []),
     ...neutrino.config.resolve.extensions.values().map(e => e.replace('.', ''))
   ])];
+  const testRegex = join(
+    basename(neutrino.options.tests),
+    `.*(_test|_spec|\\.test|\\.spec)\\.(${neutrino.options.extensions.join('|')})$`
```

Comment:
Why not `regexFromExtensions` here?

Reply:
I see why now. I keep thinking `regexFromExtensions` returns a string regex representation, not an actual regex. My bad.